package.json

```diff
@@ -93,7 +94,7 @@
     }
   },
   "lint-staged": {
-    "src/**/*.{js,json,css}": [
```

Comment:
it's not possible to use `eslint` on JSON file, right? 😂

Reply:
lol yes, need to remove that